docs/install.rst

```diff
@@ -115,6 +115,12 @@ Add Gulp Task:
           .pipe(gulp.dest('dist/css'));
     });
 
+Start using Susy:
+
+.. code-block:: sass
+
+    /* app.scss */
+    @import "_susy";
```

Comment:
Thanks for the PR! The `_` shouldn't be needed here, since Sass allows that to be implied (same as extensions). Do you mind making that change before I merge?